source/tests-integration/TradeAndReport.ts

```diff
@@ -11,21 +12,62 @@ describe("TradeAndReport", () => {
     });
     it("#tradeAndReport", async () => {
         await fixture.approveCentralAuthority();
+
+        // Create a market
         const market = await fixture.createReasonableMarket(fixture.universe, fixture.cash.address, [stringTo32ByteHex(" "), stringTo32ByteHex(" ")]);
         const actualTypeName = await market.getTypeName_();
         const expectedTypeName = stringTo32ByteHex("Market");
         expect(actualTypeName).to.equal(expectedTypeName);
 
-        const type = new BN(0);
-        const outcome = new BN(0);
-        const numShares = new BN(10000000000000);
-        const price = new BN(2150);
+        const numTicks = await market.getNumTicks_();
+
+        // Place an order
+        let type = new BN(0); // BID
+        let outcome = new BN(0);
+        let numShares = new BN(10000000000000);
+        let price = new BN(2150);
```

Comment:
Are these variables getting modified anywhere? Why are they no longer declared as constants?

Reply:
At some point I intend to add more trading in here in which case they will be modified